e2e/login.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+
+// Get data from config about projects and convert to JS object setting TS types
+
+const projects = JSON.parse(process.env.PROJECTS!) as Record<
+  string,
+  { url: string; app: string; title: string }
+>;
+
+// Create a loop to loop over all the projects and create a test for everyone
+
+for (const project of Object.keys(projects)) {
+  test(`As User, I can login to ${projects[project].title}`, async ({ page }) => {
```

Comment:
We can do it without `Object.keys` then accessing to the properties directly just `project.title`, etc